pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -582,4 +585,14 @@ public void setStatusCheckerInitialDelayInSeconds(long initialDelayInSeconds) {
   public long getPeriodicTaskInitialDelayInSeconds() {
     return ControllerPeriodicTasksConf.getRandomInitialDelayInSeconds();
   }
+
+  public void setControllerMode(String controllerMode) {
```

Comment:
Pass in HelixSetupUtils.ControllerMode instead of String?

Reply:
Done.